src/components/formControls/radio/style.ts

```diff
@@ -0,0 +1,70 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License. v. 2.0. If a copy of the MPL was not distributed with this file.
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled, { css, ThemedStyledProps } from '../../../theme'
+import { Props, Size } from './index'
+
+interface StyleProps {
+  size?: Size
+  disabled?: boolean
+  selected?: boolean
+}
+
+const getThemeColors = (p: ThemedStyledProps<Props>, selected: boolean | undefined) => {
+  let fillColor = '#696FDC'
+  let borderColor = '#EBECF0'
+
+  if (!p.disabled) {
+    borderColor = selected
+      ? p.theme.color.brandBat
+      : p.theme.color.brandBatActive
+  }
+
+  return css`
+    --radio-fill-color: ${fillColor};
+    --radio-border-color: ${borderColor};
+  `
+}
+
+export const StyledLabel = styled<StyleProps, 'label'>('label')`
+  line-height: 1.3;
+  display: flex;
+  margin-bottom: 20px;
+  color: #686978;
+  font-size: ${p => p.size === 'big' ? '16' : '14'}px;
+  cursor: pointer;
+`
+
+export const StyledInput = styled<{}, 'input'>('input')`
+  display: none;
```

Comment:
It may be better to hide this via `opacity: 0; position: absolute;` so that it's still present, for accessibility